src/loaders/lottie/tvgLottieLoader.h

```diff
@@ -39,6 +39,9 @@ class LottieLoader : public FrameModule, public Task
     float frameCnt = 0.0f;
     float frameDuration = 0.0f;
 
+    float segmentBegin = 0.0f;
```

Comment:
The segment is the animation base feature. It could be in the base class(FrameModule)